### PR TITLE
Check no version checking

### PR DESCRIPTION
**Pull request by unconst**
Two minor changes:
-- checks for no_version_checking in config creation.
-- If not command is passed to the cli we print the help

### CODE REVIEW

File: bittensor/_cli/__init__.py [hunk 1]
```diff
@@ -704,6 +707,8 @@ def check_config (config: 'bittensor.Config'):
             cli.check_help_config(config)
         elif config.command == "update":
             cli.check_update_config(config)
+        else:
```
Comment (camfairchild) [inline bittensor/_cli/__init__.py]: ?

File: bittensor/_cli/cli_impl.py [hunk 2]
```diff
@@ -41,7 +41,7 @@ def __init__(self, config: 'bittensor.Config' ):
                 config (:obj:`bittensor.Config`, `required`): 
                     bittensor.cli.config()
         """
-        if not config.no_version_checking:
+        if "no_version_checking" in config and config.no_version_checking:
```
Comment (camfairchild) [inline bittensor/_cli/cli_impl.py]: `if not config.get("no_version_checking", True):`

would be better.

**Review by camfairchild — CHANGES_REQUESTED** [review]

File: bittensor/_cli/__init__.py (hunk 1 above)
Comment (unconst) [reply bittensor/_cli/__init__.py]: lmao

File: bittensor/_cli/cli_impl.py (hunk 2 above)
Comment (unconst) [reply bittensor/_cli/cli_impl.py]: Kinda, although you need to read the docs on the get function to understand the boolean

**Comment by camfairchild** [comment]
Rebased this onto finney and resolved the changes.  

`btcli` with no other args will print the help text and `btcli [command] --no_version_checking` will skip version checks

**Review by camfairchild — APPROVED** [review]

File: bittensor/utils/__init__.py [hunk 3]
```diff
@@ -66,7 +66,7 @@ def version_checking():
     latest_version_as_int = (100 * int(version_split[0])) + (10 * int(version_split[1])) + (1 * int(version_split[2]))
 
     if latest_version_as_int > bittensor.__version_as_int__:
-        print('\u001b[31m Current Bittensor Version: {}, Latest Bittensor Version {} \n Please update to the latest version'.format(bittensor.__version__,latest_version))
+        print('\u001b[31mBittensor Version: Current {}/Latest {}\nPlease update to the latest version\u001b[0m'.format(bittensor.__version__,latest_version))
```
Comment (eduardogr) [inline bittensor/utils/__init__.py]: Should we let them know that they don't need to do it urgently?

File: bittensor/_cli/__init__.py [hunk 4]
```diff
@@ -662,11 +662,16 @@ def config(args: List[str]) -> 'bittensor.config':
             required=False
         )
 
+        # If no arguments are passed, print help text.
+        if len(args) == 0:
+            parser.print_help()
+            sys.exit()
```
Comment (eduardogr) [inline bittensor/_cli/__init__.py:668]: Thanks for this!

File: bittensor/utils/__init__.py (hunk 3 above)
Comment (camfairchild) [reply bittensor/utils/__init__.py]: I think if we make it yellow maybe.
And yes we can add like `"Please update your client when convenient"`

Comment (camfairchild) [reply bittensor/utils/__init__.py]: @eduardogr how's this?

**Review by eduardogr — APPROVED** [review]

**Pull request merged**